google/cloud/bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -243,6 +287,252 @@ void DeleteCluster(google::cloud::bigtable::InstanceAdmin instance_admin,
+  google::cloud::bigtable::ClusterId cluster_id_1(ConsumeArg(argc, argv));
+  std::string const zone_1 = ConsumeArg(argc, argv);
+  google::cloud::bigtable::ClusterId cluster_id_2(ConsumeArg(argc, argv));
+  std::string const zone_2 = ConsumeArg(argc, argv);
+
+  // [START connect_instance_admin]
+  // Create an instance admin endpoint.
+  google::cloud::bigtable::InstanceAdmin instance_admin(
+      google::cloud::bigtable::CreateDefaultInstanceAdminClient(
+          project_id, google::cloud::bigtable::ClientOptions()));
+  // [END connect_instance_admin]
+
+  // [START bigtable_check_instance_exists]
+  std::cout << std::endl << "Check Instance exists: " << std::endl;
+  auto instances = instance_admin.ListInstances();
+  auto instance_name =
+      instance_admin.project_name() + "/instances/" + instance_id.get();
+  bool instanceExists =
+      instances.end() !=
+      std::find_if(
+          instances.begin(), instances.end(),
+          [&instance_name](google::bigtable::admin::v2::Instance const& i) {
+            return i.name() == instance_name;
+          });
+  // [END bigtable_check_instance_exists]
+
+  // Create instance if does not exists
+  if (not instanceExists) {
+    // [START bigtable_create_prod_instance]
+    std::cout << std::endl << "Creating a PRODUCTION Instance: ";
+    // Create production instance
+    google::cloud::bigtable::DisplayName display_name("Put description here");
+
+    // production instance needs at least 3 nodes
+    auto cluster_config = google::cloud::bigtable::ClusterConfig(
+        zone_1, 3, google::cloud::bigtable::ClusterConfig::HDD);
+    google::cloud::bigtable::InstanceConfig config(
+        google::cloud::bigtable::InstanceId(instance_id), display_name,
+        {{cluster_id_1.get(), cluster_config}});
+    config.set_type(google::cloud::bigtable::InstanceConfig::PRODUCTION);
+
+    auto future = instance_admin.CreateInstance(config).get();
+    std::cout << " Done" << std::endl;
+    // [END bigtable_create_prod_instance]
+  } else {
+    std::cout << std::endl
+              << "Instance " << instance_id.get() << " already exists."
+              << std::endl;
+    return;
+  }
+
+  // [START bigtable_list_instance]
+  // List Instances
+  std::cout << std::endl << "Listing Instances: " << std::endl;
+  auto instances_after = instance_admin.ListInstances();
+  for (auto const& instance : instances_after) {
+    std::cout << instance.name() << std::endl;
+  }
+  // [END bigtable_list_instance]
+
+  // [START bigtable_get_instance]
+  // get Instance
+  std::cout << std::endl << "Get Instance: " << std::endl;
+  auto instance = instance_admin.GetInstance(instance_id.get());
+  std::string instance_detail;
+  google::protobuf::TextFormat::PrintToString(instance, &instance_detail);
+  std::cout << "GetInstance details : " << std::endl << instance_detail;
+  // [END bigtable_get_instance]
+
+  // [START bigtable_list_cluster]
+  // List clusters
```

Comment:
Also capitalize consistently, it is either `// List Clusters` or everywhere else we just capitalize the first word.

Reply:
Removed.